test/parallel/test-fs-promises.js

```diff
@@ -117,20 +117,33 @@ function verifyStatObject(stat) {
     verifyStatObject(stats);
 
     const newLink = path.resolve(tmpDir, 'baz3.js');
-    await symlink(newPath, newLink);
+    let symlinksSupported = true;
+    try {
+      await symlink(newPath, newLink);
+    } catch (err) {
+      // Non-admin users on Windows can't make symlinks and get EPERM.
```

Comment:
Can you check that this only happens when `common.isWindows === true`?

Reply:
Sounds good, added that check. Thanks!

Reply:
We already have [`common.canCreateSymLink()`](https://github.com/nodejs/node/tree/master/test/common#cancreatesymlink) for such checks.

Reply:
Much better, thanks!